gdgphilippines/devfest2018
# Create speaker list component #31

### Description
Updated the page to include speakers and sessions. Changed the data structure of the site though. Will update docs

### Related Issue
Fix #37 
Fix #35 
Fix #36 
Fix #34 
Fix #33 
Fix #32 
Fix #31 
Fix #30 
Fix #14 
Fix #1 

### Motivation and Context
Adds the speaker section and sessions

### How Has This Been Tested?
1. Go to site
2. Click on speakers, should see all speakers
3. Click on speaker, should go to speaker
4. Click on session, should go to session

### Types of changes


### Checklist:


- [x] My code follows the code style of this project. (delete if it is documentation)

### Reviewer Checklist:

- [x] Code follows the code style of the project (frontend/backend)
- [x] DRY Code
- [ ] Added tests to cover changes
- [ ] All tests have passed
- [x] Reviews have been done and approved
